{"uid":"4b4757e66a1912dae1a509f688f20b0f","name":"categories","children":[{"name":"Product defects","children":[{"name":"Dart test failed: temp_test Req_01: Successfully request money to single user.\n══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞═════════════════\nThe following message was thrown:\nMultiple exceptions (3) were detected during the running of the\ncurrent test, and at least one was unexpected.\n═════════════════════════════════════════════════════════════════\n","children":[{"name":"runDartTest[temp_test Req_01: Successfully request money to single user.]","uid":"c6951d877ac6ab5","parentUid":"67a35c9f7c6fcaa65ee444542bc8ca2f","status":"failed","time":{"start":1703127177353,"stop":1703127222800,"duration":45447},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":true,"parameters":[],"tags":[]}],"uid":"67a35c9f7c6fcaa65ee444542bc8ca2f"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}]}